packages/core-contracts/contracts/mocks/utils/SetUtilMock.sol

```diff
@@ -0,0 +1,72 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../utils/SetUtil.sol";
+
+contract Bytes32SetMock {
```

Comment:
Should you update the feature branch from main so that this diff isn't visible?

Reply:
fixed